.github/workflows/python.yml

```diff
@@ -5,29 +5,28 @@ name: Build Status
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.9, '3.10', '3.11', '3.12']
+        python-version: [3.9, 3.10, 3.11, 3.12]
```

Comment:
The quotes should be included, otherwise it's interpreted at `3.1`.

Reply:
Okay cool. Let me do it

Reply:
I have made the necessary changes. The CI pipeline should pass now